test/runtests.jl

```diff
@@ -1,5 +1,35 @@
 using TiledIteration, OffsetArrays
 using Test
+using Documenter
+
+if VERSION < v"1.6-"
+    Documenter.doctest(TiledIteration)
+    # Version restriction can be lifted, when
+    # filters can be passed to `doctest`
+    # See https://github.com/JuliaDocs/Documenter.jl/pull/1435
+    #
+    # doctestfilters = [
+    #     r"{([a-zA-Z0-9]+,\s?)+[a-zA-Z0-9]+}",
+    #     r"(Array{[a-zA-Z0-9]+,\s?1}|Vector{[a-zA-Z0-9]+})",
+    #     r"(Array{[a-zA-Z0-9]+,\s?2}|Matrix{[a-zA-Z0-9]+})",
+    # ]
+    # Documenter.doctest(TiledIteration, doctestfilters = doctestfilters)
+end
+
+
+@testset "TileIterator small examples" begin
+    titr = @inferred TileIterator((1:10,), RelaxLastTile((3,)))
+    @test titr == [(1:3,), (4:6,), (7:9,), (10:10,)]
+
+    titr = @inferred TileIterator((1:10,), RelaxStride((3,)))
+    @test titr == [(1:3,), (3:5,), (6:8,), (8:10,)]
+
+    titr = @inferred TileIterator((1:4,), RelaxStride((2,)))
+    @test titr == [(1:2,), (3:4,)]
+
+    titr = @inferred TileIterator((1:4,), RelaxLastTile((2,)))
+    @test titr == [(1:2,), (3:4,)]
+end
```

Comment:
Should also add some test cases for 2d cases; those are currently live in doctest but it'd better to also have them here.

Reply:
done